source/features/search-filters.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {XIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import SearchQuery from '../github-helpers/search-query';
+import {getUsername} from '../github-helpers';
+
+function init(): void {
+	const items = [
+		['Forks', 'fork:true'],
+		['Private', 'is:private'],
+		['Yours', `user:${getUsername()}`]
```

Comment:
```suggestion
		['Yours', `user:@me`]
```

Reply:
For some reason GitHub doesn't show code search results when using `user:@me`:

<details>
<summary><code>user:@me</code></summary>

![image](https://user-images.githubusercontent.com/44045911/111408437-43b39080-8710-11eb-932c-b9e63e9e8a85.png)

</details>

<details>
<summary><code>user:username</code></summary>

![image](https://user-images.githubusercontent.com/44045911/111408484-529a4300-8710-11eb-9c6c-73263d6abd20.png)

</details>